charts/ratify/values.yaml

```diff
@@ -71,12 +73,11 @@ oras:
     awsEcrBasicEnabled: false
 
 provider:
-  auth: ""                  # tls, mtls
+  auth: "tls"                  # auth must be tls or mtls
```

Comment:
can you use clientCA to infer mTLS intend and you can eliminate this field?

Reply:
good call, I'll update it.

Reply:
Updated!